rtengine/dcraw.cc

```diff
@@ -10133,8 +10158,12 @@ void CLASS identify()
         width = raw_width = 6016;
         height = raw_height = 4014;
     } else if (!strcmp(model, "X-Pro3") || !strcmp(model, "X-T3") || !strcmp(model, "X-T30") || !strcmp(model, "X-T4") || !strcmp(model, "X100V") || !strcmp(model, "X-S10")) {
-        width = raw_width = 6384;
-        height = raw_height = 4182;
+        raw_width = 6384;           // RT
+        raw_height = 4182;          // RT
+        if (!read_crop.crop_mode) { // RT
+            width = 6384;           // RT
```

Comment:
Though semantically the same, the more expressive term would be `width = raw_width;`, wouldn't it?